examples/examples.js

```diff
@@ -33,6 +33,89 @@ const Random = {
         return arrayOrString[index];
     },
     
+    uuid: function() {
```

Comment:
This should use `crypto.randomUUID` on the server and web crypto randomUUID on the client